config/default.schema.json

```diff
@@ -231,6 +231,10 @@
 				}
 			}
 		},
+		"NO_COLOR": {
```

Comment:
Shouldn't we name the variable so that we can see that it belongs to logging? `NO_COLOR` is way too general IMHO.